### PR TITLE
Create phpunit-sqlite.yml

### PR DESCRIPTION
**Pull request by come-nc**
Importing workflows from nextcloud/.github

### CODE REVIEW

**Comment by come-nc** [comment]
Problem here is that server master is now 25 and this app is flagged as compatible with up to 24 I think.
But stable24 is not branched off here it seems so not sure if ok to change info.xml to allow 25.

**Comment by come-nc** [comment]
> Bump was not merged yet #563

Yes I get that but the trouble is that the workflows expects the master branch to be able to install on the server master branch.
And my question was can we merge this version bump here in master or should we branch out a stable24 branch before? (And I never branched out an app before so I’m not sure of the right way to do it, if there is anything to know or if it’s just creating the branch)

File: composer.json [hunk 1]
```diff
@@ -11,5 +11,14 @@
             "email": "jus@bitgrid.net"
         }
     ],
-    "require": {}
+    "require": {},
+	"scripts": {
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l",
+		"cs:check": "php-cs-fixer fix --dry-run --diff",
+		"cs:fix": "php-cs-fixer fix",
+		"psalm": "psalm --threads=1",
```
Comment (nickvergessen) [inline composer.json:19]: Missing from require dev?

Comment (nickvergessen) [inline composer.json:19]: Missing from require dev?

File: composer.json [hunk 2]
```diff
@@ -11,5 +11,14 @@
             "email": "jus@bitgrid.net"
         }
     ],
-    "require": {}
+    "require": {},
+	"scripts": {
+		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l",
+		"cs:check": "php-cs-fixer fix --dry-run --diff",
```
Comment (nickvergessen) [inline composer.json:17]: Missing from require dev?

Comment (nickvergessen) [inline composer.json:17]: Missing from require dev?

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**